.github/workflows/ci.yml

```diff
@@ -414,7 +414,9 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: 'deno-${{ matrix.os }}-${{ matrix.arch }}-${{ github.event.number }}'
-          path: target/release/deno
+          path: |
+            target/release/deno
+            target/release/denort
```

Comment:
Is this on purpose? If so, maybe we could upstream it in a separate PR so it's easier to revert?

Reply:
I'll remove from this PR (I needed to have denort binary built for linux so that I could test it)